types/eval-estree-expression/eval-estree-expression-tests.ts

```diff
@@ -0,0 +1,24 @@
+import { evaluate } from "./index";
```

Comment:
Tests should import as if they were a user of the package:
```suggestion
import { evaluate } from "eval-estree-expression";
```